en/guide/telemetry.md

```diff
@@ -189,6 +189,7 @@ Similarly, you can use the asynchronous method and block
     // Now ready to arm
 }
 ```
+> **Note** Make sure to unsubscribe from the callback after it suceed using `telemetry->health_all_ok_async(nullptr)`. Otherwise you will run into an error because `prom->set_value()` is called more than once.

```

Comment:
spelling is "succeed".

Reply:
@JonasVautherin Anything to add to this?

Reply:
Don't need to add it in the code example IMO. The example is working while it is not called more than once

Reply:
I need to do a new commit to propose your new note ?

Reply:
@Katawann You would have but I did it for you now (I would have done this first time, but I wanted to make sure it still addresses your concern).

@JonasVautherin @julianoes Can either of you confirm that you're now happy with this PR?